deltachat-ios/AppDelegate.swift

```diff
@@ -103,10 +103,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         logger.info("---- foreground ----")
         appIsInForeground = true
         startThreads()
-
         if reachability.connection != .none {
             self.dcContext.maybeNetwork()
         }
+
+        let userDefaults = UserDefaults.shared
+        if userDefaults?.bool(forKey: UserDefaults.hasExtensionAttemptedToSend) ?? false {
```

Comment:
How about:
`if let userDefaults = UserDefaults.shared, userDefaults.bool(forKey ....) { /* if block*/}`

Reply:
yes that's nicer, thanks

Reply:
fixed